pkg/component/monitoring/component.go

```diff
@@ -53,6 +55,8 @@ var (
 	//go:embed charts/seed-monitoring/charts/core
 	chartCore     embed.FS
 	chartPathCore = filepath.Join("charts", "seed-monitoring", "charts", "core")
+
+	managedResourceNamePrometheus = "shoot-core-prometheus"
```

Comment:
Do we not need to ignore the prometheus resources in `shoot-core` managed resource? How will migration look like here? `shoot-core` MR will try to delete the resource and `shoot-core-prometheus` MR will try to apply the same resource, is this not an issue?

Reply:
No migration needed here - I renamed the resources, so the old RBAC will disappear (due to `shoot-core` `ManagedResource` reconciliation), and the new RBAC will appear in parallel.

Reply:
Ok, I didn't notice the resource name change.